dashboard/Dockerfile

```diff
@@ -20,9 +20,11 @@ RUN apt-get update && \
     && rm -rf /var/lib/apt/lists/*
 COPY package.json /dashboard/
 COPY yarn.lock /dashboard/
-RUN cd /dashboard && npm i yarn && npx yarn install --ignore-scripts
+COPY typings.json /dashboard/
+COPY package-lock.json /dashboard/
```

Comment:
hello, why do we have the need of package-lock.json as we're using yarn ?

Reply:
@benoitf it's being discussed also in https://github.com/eclipse/che/pull/12923 IMHO better approach is to get rid of `npm i yarn` so that the `package-lock.json` won't be generated at all

Reply:
@ppitonak yes but I prefer what is done in https://github.com/eclipse/che/pull/12923

Reply:
I'm confused... do you mean the suggestion by @monaka ?

Reply:
not using package-lock and npm, but using only yarn

Reply:
@benoitf  In this PR I lived an old solution, just fix needed yarn version in package-lock.json.  
Ok. I will change it(upgrade the yarn version in the image)

Reply:
done